pyproject.toml

```
@@ -58,7 +58,7 @@ platforms = [ "linux-64",]
 [tool.pixi.dependencies]
 python = "*"
 pip = "*"
-mantidworkbench = ">=6.12"
+mantidworkbench = ">=6.13"
```

```suggestion
mantidworkbench =">=6.13.0,<6.14.0"
```
This will avoid inadvertently deploying mantid 6.14 to production